beaconApp/src/main/java/com/pulselink/beacon/BeaconApp.kt

```
@@ -3,14 +3,19 @@ package com.pulselink.beacon
 import android.app.Application
 import com.google.android.gms.ads.MobileAds
 import androidx.lifecycle.ProcessLifecycleOwner
+import com.pulselink.beacon.data.SmsSyncManager
 
 class BeaconApp : Application() {
     private lateinit var appOpenManager: AppOpenAdManager
+    lateinit var smsSyncManager: SmsSyncManager
```

detekt (1.23.8)

[warning] Usages of lateinit should be avoided.

(detekt.potential-bugs.LateinitUsage)